applications/wide_baseline_tracker_test.cpp

```diff
@@ -0,0 +1,169 @@
+ *  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+ *  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  Author:
+ *
+ *  Luiz Felipe Maciel Correia
+ *  Marcos Henrique Marcone
+ *  Bruno Silva
+ */
+
+#include <cstdio>
+#include <cstdlib>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+
+#include <rgbd_loader.h>
+#include <wide_baseline_tracker.h>
+#include <config_loader.h>
+#include <event_logger.h>
+#include <common_types.h>
+
+using namespace std;
+using namespace cv;
+
+void draw_last_track(Mat& img, const vector<Point2f> prev_pts, const vector<Point2f> curr_pts, bool is_kf);
+void draw_tracks(Mat &img, const vector<Tracklet> tracklets);
+
+/**
+ * This program shows the use of tracking by detection algorithm.
+ * @param .yml config. file (from which index_file is used)
+ */
+int main(int argc, char **argv)
+{
+	EventLogger& logger = EventLogger::getInstance();
+	logger.setVerbosityLevel(pcl::console::L_INFO);
+	
+	RGBDLoader loader;
+	string index_file;
+	int log_stats;
+	Mat frame, depth, current_frame, previous_frame;
+	Ptr<cv::FeatureDetector> feature_detector;
+	Ptr<cv::DescriptorExtractor> descriptor_extractor;
+	Ptr<cv::DescriptorMatcher> descriptor_matcher;
+	
+	if (argc != 2)
+	{
+		logger.print(pcl::console::L_INFO, "[wide_baseline_tracker_test.cpp] Usage: %s <path/to/config_file.yaml>\n", argv[0]);
+		exit(0);
+	}
+
+	ConfigLoader param_loader(argv[1]);
+	param_loader.checkAndGetString("index_file", index_file);
+	param_loader.checkAndGetInt("log_stats", log_stats);
+	param_loader.checkAndGetFeatureDetector("feature_detector", feature_detector);
+	param_loader.checkAndGetDescriptorExtractor("descriptor_extractor", descriptor_extractor);
+	param_loader.checkAndGetDescriptorMatcher("descriptor_matcher", descriptor_matcher);
+	
+	WideBaselineTracker wide_baseline_tracker(feature_detector, descriptor_extractor, descriptor_matcher, log_stats);
+
+	loader.processFile(index_file);
+	
+	//Track points on each image
+	for (int i = 0; i < loader.num_images_; i++)
+	{
+		loader.getNextImage(frame, depth);
+
+		double el_time = (double) cvGetTickCount();
+		bool is_kf = wide_baseline_tracker.track(frame);
+		el_time = ((double) cvGetTickCount() - el_time)/(cvGetTickFrequency()*1000.0);
+		logger.print(pcl::console::L_INFO,"[klt_tracker_test.cpp] INFO: Tracking time: %f ms\n", el_time);
+		
+		frame.copyTo(current_frame);
+
+		draw_last_track(frame, wide_baseline_tracker.prev_pts_, wide_baseline_tracker.curr_pts_, is_kf);
+		//draw_tracks(frame, wide_baseline_tracker.tracklets_);
+
+		if (i > 0)
+		{
+			Mat img_matches;
+			drawMatches(current_frame, wide_baseline_tracker.curr_KPs_, previous_frame, wide_baseline_tracker.prev_KPs_, wide_baseline_tracker.matches_, img_matches, Scalar::all(-1),
```

Comment:
Change curr_KPs_ to curr_kpts_; curr_good_Pts_ to curr_good_pts_

Reply:
Was solved in the commit 429d944b5468eff2331c6fa37d37ee593979f774.